tools/inference_engine/src/impl/nodes/matmul.cpp

```diff
@@ -4,10 +4,11 @@
 #include <iostream>
 
 
-void inference_engine::GpuMatMul::fuse_with(const GpuActivation* activation)
+void inference_engine::GpuMatMul::fuse_with(const std::vector<GpuActivation*>& activations)
```

Comment:
Could this method be defined in base class?
Also I think it should accept single operator (please have a look at above comments)